crates/ruff_python_formatter/src/expression/mod.rs

```diff
@@ -199,6 +201,12 @@ impl Format<PyFormatContext<'_>> for MaybeParenthesizeExpression<'_> {
             OptionalParentheses::Always => {
                 expression.format().with_options(Parentheses::Always).fmt(f)
             }
+            OptionalParentheses::Never | OptionalParentheses::Multiline
+                if *parenthesize == Parenthesize::ForceIfBreaks =>
```

Comment:
Is the path `OptionalParentheses::Multiline` ever taken? Doesn't it already pass the aboth multiline case (because `parenthesize != Parenthesize::IfRequired` is true

Reply:
Good call.